symfony/phpunit-bridge/3.3/manifest.json

```diff
@@ -8,6 +8,5 @@
     "gitignore": [
         ".phpunit",
         "/phpunit.xml"
-    ],
-    "aliases": ["phpunit", "simple-phpunit"]
```

Comment:
I would still keep the `simple-phpunit` alias

Reply:
fixed